amz_review_scraper/models.py

```diff
@@ -12,6 +12,9 @@ class Item(db.Model):
     def save(self):
```

Comment:
the comment on Item is absolute now, you did update it!

Reply:
lol yup, i decided it was an Item not an Items